certified-connectors/RescoReports/apiDefinition.swagger.json

```diff
@@ -185,7 +186,7 @@
       "get": {
         "summary": "List reports",
         "description": "Get list of records",
-        "operationId": "Report-List",
+        "operationId": "reportList",
```

Comment:
@JuroJ all operationIds should be in Camel Case. Please change to "ReportList". It should start with a capital letter. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).
Please fix all operationIds in the swagger file.